mirage/views/collection-submission-action.ts

```diff
@@ -0,0 +1,40 @@
+import { HandlerContext, NormalizedRequestAttrs, Request, Schema } from 'ember-cli-mirage';
+import CollectionSubmissionAction,
+{ CollectionSubmissionActionTrigger } from 'ember-osf-web/models/collection-submission-action';
+import { CollectionSubmissionReviewStates } from 'ember-osf-web/models/collection-submission';
+
+export function createSubmissionAction(this: HandlerContext, schema: Schema, request: Request) {
+    const attrs = this.normalizedRequestAttrs('collection-submission-action') as
+        Partial<NormalizedRequestAttrs<CollectionSubmissionAction>>;
+    const submissionId = request.params.submissionID;
+    const userId = schema.roots.first().currentUserId;
+    let collectionSubmissionAction;
+    if (userId && submissionId) {
+        const currentUser = schema.users.find(userId);
+        const submission = schema.collectionSubmissions.find(submissionId);
+        const { trigger, comment } = attrs as any; // cast attrs to any because `actionTrigger` does not exist on type
+        collectionSubmissionAction = schema.collectionSubmissionActions.create({
```

Comment:
Can you `schema.create('collection-submission-actions', {` here? It probably doesn't matter, since you're filling in all the info below, but I thiiiiink if you can do that it'll use mirage to create the object rather than ember-data, which would mean it would use factories in case that's a problem in the future. `<time passes />` Checking through some code, it looks like you'd need the server here, not just the schema, so that may not be an option.